addons/test_import_export/tests/test_import.py

```diff
@@ -12,15 +12,17 @@
 from odoo.tools.misc import file_open
 from odoo.addons.base_import.models.base_import import ImportValidationError
 
-ID_FIELD = {
-    'id': 'id',
-    'name': 'id',
-    'string': "External ID",
 
-    'required': False,
-    'fields': [],
-    'type': 'id',
-}
+def get_id_field(model_name):
+    return {
+        'id': 'id',
+        'name': 'id',
+        'string': "External ID",
```

Comment:
I see it was like this before, but `id` is an internal id.

Reply:
> but id is an internal id.

Not for the import :disappointed: (`.id` is actually the internal id). Yes, it was very confusing for me too.